dao/story.dao.server.js

```diff
@@ -11,33 +10,15 @@ findStoryByPlaceID = (placeID, limit, page) => storyModel.find({place_ids:{$elem
 
 findStoryByTitle = (title, limit, page) => storyModel.find({story_title:{$regex: title,$options:'i'}}).skip((page-1)*limit).limit(limit);
 
-createStory = story => {
-    story.story_id=ObjectID().str;
-    return storyModel.create(story);
-};
+createStory = story => storyModel.create(story);
 
 deleteStory = (storyId) => storyModel.deleteOne({story_id: storyId});
 
-updateStory = (storyID, story) => storyModel.update({story_id: storyID}, {$set: story});
-
-const likeStory = (story, userID) => {
-    if(!story.liked_by_users.includes(userID)){
-        story.liked_by_users.push(userID);
-        return story;
-    } 
-        return story;
-    
-}
-
-const unlikeStory = (story, userID) => {
```

Comment:
How do we unlike the story if we don't have an endpoint for unliking a story as this PR removes the unlike part?

Reply:
Hi, Neha. Just considering from pure DB operation in DAO layer. They are just update operations. It's routes's responsibility to add or remove a user in the Map or array, then send this result to DAO layer.